less/mixins.less

```diff
@@ -1 +1,26 @@
-// Mixins in here
\ No newline at end of file
+// Mixins in here
+.make-btn(@color:white,@bg-color:@button-bg,@height:4.5rem,@width:20rem,@f-size:1.8rem) {
+    display: inline-block;

```

Comment:
You can add a `cursor: pointer;` to turn the mouse from an arrow into that little hand. Tiny tweak, but it can help '' '